src/DropdownMenu.tsx

```diff
@@ -102,6 +143,21 @@ const DropdownMenu: DropdownMenu = React.forwardRef(
     const prefix = useBootstrapPrefix(bsPrefix, 'dropdown-menu');
     const [popperRef, marginModifiers] = usePopperMarginModifiers();
 
+    const alignClasses: string[] = [];
+    if (align) {
+      if (typeof align === 'object') {
+        DEVICE_SIZES.forEach((brkPoint) => {
```

Comment:
**suggestion:** it would be cleaner to do `Object.entries(align).forEach(([breakpoint, direction]) => ...)`

the `DEVICE_SIZES` here seems extraneous

Reply:
Thanks.  I used `Object.keys` instead for IE11 compatibility.